hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/store/AbstractNearCacheRecordStore.java

```diff
@@ -279,7 +279,7 @@ public V get(K key) {
                     return null;
                 }
                 if (isRecordExpired(record)) {
-                    remove(key);
+                    invalidate(key);
```

Comment:
This will be counted as an invalidation in the stats. Is that ok?

Reply:
Actually there is no clear separation between invalidation and removal. It makes things complicated. Since invalidations is only for internal test usage, i see having one method is ok.